lib/gui/components/safe-webview/safe-webview.js

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2017 resin.io
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const angular = require('angular');
+const {
+  react2angular
+} = require('react2angular');
+const MODULE_NAME = 'Etcher.Components.SafeWebview';
+const SafeWebview = angular.module(MODULE_NAME, []);
+
+SafeWebview.component('safeWebview',
```

Comment:
Given that the whole component is just one file, can we join everything as `lib/gui/components/safe-webview.js`?